templates/bot/ts/workflow/README.md

```diff
@@ -0,0 +1,246 @@
+# Responding to card actions
+
+The Adaptive Card action handler feature enables the app to respond to adaptive card actions that triggered by end users to complete a sequential workflow. When user gets an Adaptive Card, it can provide one or more buttons in the card to ask for user's input, do something like calling some APIs, and then send another adaptive card in conversation to response to the card action.
+
+This application is built with the [Microsoft Bot Framework](https://dev.botframework.com/) running on a restify server running on App Service along with the [Azure Bot Service](https://azure.microsoft.com/services/bot-services/).
+
+
+# Getting Started
+
+Run your app with local debugging by pressing `F5` in VSCode. Select `Debug (Edge)` or `Debug (Chrome)`.
+
+**Congratulations**! You are running an application that can now send respond to a chat command in Teams.
+
+>
+> **Prerequisites**
+>
+> To run locally, you will need:
+>
+> - `Node.js` installed locally (recommended version: 14)
+> - An [M365 account for development](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts)
+>
+> **Note**
+>
+> Your app can be installed into a team, or a group chat, or as personal app. See [Installation and Uninstallation](https://aka.ms/teamsfx-command-response#customize-installation).
+>
+
+# Understanding the code
+
+This section walks through the generated code. The project folder contains the following:
+
+| Folder | Contents |
+| - | - |
+| `.fx` | Project level settings, configurations, and environment information |
+| `.vscode` | VSCode files for local debug |
+| `bot` | The source code for the command and response Teams application |
+| `templates` | Templates for the Teams application manifest and for provisioning Azure resources |
+
+The core command-response implementation is in `bot` folder.
+
+The following files provide the business logic for command and response bot. These files can be updated to fit your business logic requirements. The default implementation provides a starting point to help you get started.
```

Comment:
> The following files provide the business logic for command and response bot.

update the README as well?

Reply:
Updated. This is a draft version, and will work with @MuyangAmigo to consolidate the document.